src/components/team-session/component.tsx

```diff
@@ -126,7 +145,22 @@ const TeamSession: React.FC<Props> = ({ team }) => {
             <li />
           </ul>
 
-          <footer className="flex items-end p-2 border-t border-white border-opacity-10">
+          <footer className="flex flex-col items-end p-2 border-t border-white border-opacity-10">
+            {emojiPickerState && (
+              <Picker
+                theme="dark"
+                onSelect={handleAddReaction}
+                style={{ position: 'absolute', bottom: 0, left: "3rem", zIndex: 10 }}
```

Comment:
Maybe there could be some space between the bottom of the picker and the screen, right now it feels forced to the bottom. Maybe try using `bottom: "3rem"`as well?

Also you should use tailwind classes instead of style I think. Let me know if you need help with that.

Reply:
Picker does not read external style, than I would need to add an additonal div, and nest styling

Reply:
I think it would look better if the bottom left corner of the picker was in the same position as the left sidebar. Maybe wrap it with a div and use similar tailwind classes as the sidebar to get it into the same position?

Reply:
Why rem, we dont use it anywhere else?

Reply:
You use it on the same line. But choose something that makes sense.